docs/manual/usage.rst

```diff
@@ -95,8 +95,8 @@ add the WARC to a new collection and start pywb:
 
       docker pull webrecorder/pywb
       docker run -e INIT_COLLECTION=my-web-archive -v /pywb-data:/webarchive \
-         -v /path/to:/source webrecorder/pywb wb-manager add default /path/to/my_warc.warc.gz
-      docker run -p 8080:8080 -v /pywb-data/:/webarchive wayback
+         -v /path/to:/source webrecorder/pywb wb-manager add my-web-archive /source/my_warc.warc.gz
+      docker run -p 8080:8080 -v /pywb-data/:/webarchive webrecorder/pywb
```

Comment:
```suggestion
      docker run -p 8080:8080 -v /pywb-data/:/webarchive webrecorder/pywb wayback
```

It will work without this but to clarify intent I think it's good to include the `wayback` command here.

Reply:
fair enough!